src/disco/css/Addon.scss

```diff
@@ -53,20 +53,7 @@ $addon-padding: 20px;
     }
   }
 
-  .Addon-install-button {
```

Comment:
Are these redundant due to markup changes, or is this just controlled elsewhere?